teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>Why has the molecule argument the default <strong>None</strong>? Ahh, in case there is no small molecule in the system :)</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>yes exactly, so it is usable for simple molecules as well</p>